models-update.patch

```diff
@@ -0,0 +1,628 @@
+From e9355552893c948088021b32a1fd6e472d362489 Mon Sep 17 00:00:00 2001
+From: Gilbert Turner <grturner@5x5code.com>
+Date: Wed, 22 Apr 2020 09:35:30 -0400
+Subject: [PATCH] Model Updates: Users & Addresses Created Address model, and
+ refactored Users and DistanceServiceImpl to update
```

Comment:
I don't exactly know where this file came from, but it should not be added to the repository.